### ISSUE TITLE
Windows users: deleting this is impossible :D

### ISSUE THREAD

**Issue by jenskooij** [incentro/generator-frontend-incubator#16]
After trying out the incubator just to try it out and see how it works, i wanted to delete the folder which I had created for it. 
Alas: 
![xku4ajf 1](https://cloud.githubusercontent.com/assets/12395129/13848033/488d5b8c-ec50-11e5-9ecf-bb32dc3acf19.png)

After doing some research I found out that Windows has a set maxlength for path names of 260 characters (See https://msdn.microsoft.com/en-us/library/aa365247.aspx#maxpath). With the complex structure in `node_modules`, with nested dependencies and so on, you easily get a structure you can now longer delete from explorer in Windows (neither from the commandline by the way).

I stumpled upon a nice little tool for file deletion called West Wind DeleteFiles (https://github.com/RickStrahl/DeleteFiles). I had to use this to be able to delete the folder on my Windows machine.

![delete](https://cloud.githubusercontent.com/assets/12395129/13848169/dd909fe6-ec50-11e5-90f1-547770d637db.png)

### In conclusion
Maybe it wasn't neccessary to create an issue for this specifically, but maybe it's worth noting in the readme.md or somewhere, that installing the incubator results in a un-deleteable folder on Windows xD.

**Comment by jenskooij** [comment]
As @rburgt told me, this has to do with my ancient npm version (I had 2.14.20). Turns out this doesn't happen with newer npm versions.

**Comment by rburgt** [comment]
We should implement a way of warning users about this. This should help:
http://www.marcusoft.net/2015/03/packagejson-and-engines-and-enginestrict.html

**Comment by kaspereden** [comment]
We defenitely should indeed. Will dive into this tomorrow

**Comment by kaspereden** [comment]
I am not sure what version of node is required but since it works in mine I've added those. Also made the tool unavailable when you do not have the correct version of node.
